SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -1859,7 +1859,7 @@ HashPeImage (
   SectionHeader = NULL;
   Status        = FALSE;
 
-  if (HashAlg != HASHALG_SHA256) {

```

Comment:
Do you really need this change?

Reply:
We don't I'll see if I can drop the change in one of the earlier commits